### ISSUE TITLE
tensorflow.contrib.slim padding='same'

### ISSUE THREAD

**Issue by Jo-won** [Jo-won/Me-Mo#27]
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/layers.py

-> max_pool2d

https://stackoverflow.com/questions/53819528/how-does-tf-keras-layers-conv2d-with-padding-same-and-strides-1-behave
